src/lib/containers/markdown/MarkdownEditor.tsx

```diff
@@ -42,18 +120,13 @@ export const MarkdownEditor: React.FunctionComponent<MarkdownEditorProps> = ({
         </div>
         {currentTab === MarkdownEditorTabs.WRITE && (
           <div className="MarkdownEditorControlsToolbar">
-            <button>
-              <IconSvg options={{ icon: 'title' }} />
-            </button>
-            <button>
-              <IconSvg options={{ icon: 'bold' }} />
-            </button>
-            <button>
-              <IconSvg options={{ icon: 'italic' }} />
-            </button>
-            <button>
-              <IconSvg options={{ icon: 'code' }} />
-            </button>
+            {commandList.map(type => {
+              return (
+                <button key={type} onClick={() => handleCommands(type)}>
```

Comment:
If you run into type issues where it conflicts with the Icon type, you could use a type cast